packages/netlify-cms-core/src/__tests__/backend.spec.js

```diff
@@ -454,4 +454,125 @@ describe('Backend', () => {
       );
     });
   });
+
+  describe('combineMultiContentEntries', () => {
```

Comment:
I like the tests! `mutiple` should be `multiple` across this file.

Reply:
*`multi` should be `multiple`